lang/csharp/src/apache/main/CodeGen/CodeGen.cs

```diff
@@ -989,29 +990,34 @@ internal static string getType(Schema schema, bool nullible, ref bool nullibleEn
         /// <returns>
         /// schema that is nullable.
         /// </returns>
+        /// <exception cref="System.ArgumentNullException">schema - UnionSchema can not be null.</exception>
+        [Obsolete("Use GetNullableType. This method will be deprecated in a future release.")]
```

Comment:
Am I wrong or `[Obsolete]` in C# is the same as `@Deprecated` in Java ?
I.e. by adding `[Obsolete]` the following statement is already deprecated?
If I am correct then the message should say `This method is deprecated and it will be removed in a future release.`